tutorials/applications/QuantumEvolutionKernel_Tutorial.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
```

Comment:
<p>This list is not adequately formatted on the online docs, you might need a space before you start listing points?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>